x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/geo_threshold/query_builder/index.tsx

```diff
@@ -100,6 +120,12 @@ export const GeoThresholdAlertTypeExpression: React.FunctionComponent<AlertTypeP
       }
     }
   };
+  const [tempIndexQuery, setTempIndexQuery] = useState<Query>(
```

Comment:
consider to remove the `temp` prefix and just rename this `indexQuery`.

I've tried to find for similar examples on naming conventions for hooks, and this seems pretty conventional stuff here, similar to `this.state`. I think the `temp` distracts from that.

Reply:
We already have `indexQuery` in the alert params, we don't want to overlap with that. This is the temp version of the variable kept in local state as a user types. A validated `tempIndexQuery` gets "promoted" to an `alertParams` > `indexQuery`. Having said that, I have no problem renaming. How about `indexQueryInput`?